signac/__main__.py

```diff
@@ -153,9 +153,12 @@ def find_with_filter(args):
             return args.job_id
 
     project = get_project()
-    f = parse_filter_arg(args.filter)
-    df = parse_filter_arg(args.doc_filter)
-    return project._find_job_ids(filter=f, doc_filter=df)
+    filter_ = parse_filter_arg(args.filter) or {}
+    if args.doc_filter:
+        filter_.update(_add_prefix("doc.", parse_filter_arg(args.doc_filter)))
```

Comment:
Should we try to handle conflicts here? If yes, then I would suggest to have a canonical helper function for handling this logic and use it everywhere else as well.

Reply:
I think it's fine for users to have `doc_filter` override unified query parameters. I don't think writing more logic / testing provides a benefit to users when this is deprecated anyway.